src/CommonITILSatisfaction.php

```diff
@@ -212,6 +215,20 @@ public function prepareInputForUpdate($input)
             }
         }
 
+        if (array_key_exists('satisfaction', $input) && $input['satisfaction'] >= 0) {
```

Comment:
Should probably done also in `prepareInputForAdd`.

Reply:
The inquest is created by the backend when the ticket is closed, users then update the values.